plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/TestSqlServerConnectorTest.java

```diff
@@ -151,6 +155,22 @@ public void testInsertWriteBulkinessWithTimestamps(String timestampType)
         }
     }
 
+    @Test(dataProvider = "testTableNameDataProvider")
+    public void testCreateTableWithSpecialCharacterName(String tableName)
+    {
+        if (!hasBehavior(SUPPORTS_CREATE_TABLE)) {
```

Comment:
you know whether SQL Server supports create table, so this condition is redundant